src/base-cookie.js

```diff
@@ -47,6 +67,19 @@ const set = (name, value, opts) => {
   document.cookie = str;
 };
 
+const getLastEventTime = (cookie) => {
+  const strValue = cookie.split('.')[Constants.LAST_EVENT_TIME_INDEX];
+  return parseInt(strValue, 32) || 0;
+};
+
+const sortByEventTime = (cookies) => {
```

Comment:
super super small nit (and probably not worth addressing): This will also sort the passed in cookie array. Thoughts on creating a new one? (probably we shouldn't even if this is only internal)